config/components/rbac/role.yaml

```diff
@@ -52,6 +52,7 @@ rules:
   - batch
   resources:
   - jobs
+  - jobs/finalizers
```

Comment:
This is auto-generated by kubebuilder, you should update the annotations on job controller. Located at `pkg/controller/workload/job/job_controller.go`. Also as less verbs as we can for security.

Reply:
Ah yes, thank you I have updated the controller annotations and reduced the set of verbs required